systemd/jenkins.service

```diff
@@ -0,0 +1,142 @@
+#
+# This file is managed by systemd(1). Do NOT edit this file manually!
+# To override these settings, run:
+#
+#     systemctl edit @@ARTIFACTNAME@@
+#
+# For more information about drop-in files, see:
+#
+#     https://www.freedesktop.org/software/systemd/man/systemd.unit.html
+#
+
+[Unit]
+Description=Jenkins Continuous Integration Server
+Requires=network.target
+After=network.target
+
+[Service]
+Type=notify
+NotifyAccess=main
+ExecStart=/usr/bin/@@ARTIFACTNAME@@
+Restart=on-failure
+SuccessExitStatus=143
+
+# Configures the time to wait for start-up. If Jenkins does not signal start-up
+# completion within the configured time, the service will be considered failed
+# and will be shut down again. Takes a unit-less value in seconds, or a time span
+# value such as "5min 20s". Pass "infinity" to disable the timeout logic.
+#TimeoutStartSec=90
+
+# Unix account that runs the Jenkins daemon
+# Be careful when you change this, as you need to update the permissions of
+# $JENKINS_HOME, $JENKINS_LOG, and (if you have already run Jenkins)
+# $JENKINS_WEBROOT.
+User=@@ARTIFACTNAME@@
+Group=@@ARTIFACTNAME@@
+
+# Directory where Jenkins stores its configuration and workspaces
+Environment="JENKINS_HOME=/var/lib/@@ARTIFACTNAME@@"
+WorkingDirectory=/var/lib/@@ARTIFACTNAME@@
+
+# Location of the Jenkins WAR
+#Environment="JENKINS_WAR=/usr/share/java/@@ARTIFACTNAME@@.war"
```

Comment:
`EnvironmentFile=-/etc/sysconfig/jenkins` can be used for optional parameters 
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=

rpm - "%{_sysconfdir}/sysconfig/${SERVICE}"
deb - "%{_sysconfdir}/default/${SERVICE}"

https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/
https://www.debian.org/doc/manuals/maint-guide/dother.en.html#initd

Reply:
> `EnvironmentFile=-/etc/sysconfig/jenkins` can be used for optional parameters

But this is not generally recommended.

Reply:
Would you please give me the link on where I can find this recommendation on systemd man page?